requirements/edx/github.txt

```diff
@@ -50,7 +50,9 @@ git+https://github.com/edx/edx-lint.git@8bf82a32ecb8598c415413df66f5232ab8d974e9
 -e git+https://github.com/edx/xblock-utils.git@581ed636c862b286002bb9a3724cc883570eb54c#egg=xblock-utils
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 -e git+https://github.com/edx/edx-reverification-block.git@2ff0d21f6614874067168bd244e68d8215041f3b#egg=edx-reverification-block
-git+https://github.com/edx/ecommerce-api-client.git@0.3.0#egg=ecommerce-api-client==0.3.0
+# NOTE edx fork of slumber is a dependency of ecommerce-api-client but cannot be installed automatically by that client.
+#git+https://github.com/edx/slumber.git@0.7.1-decode-utf8#egg=slumber==0.7.1-decode-utf8
```

Comment:
this is not meant to be commented, had it this way locally for debugging, will remove.